### ISSUE TITLE
Voiceover reads checkbox labels twice

### ISSUE THREAD

**Issue by 36degrees** [alphagov/reported-bugs#6]
**Upstream bug:** https://bugs.webkit.org/show_bug.cgi?id=191743
**Date:** 2018-11-16
**Reported by:** Oliver Byford
**Related to:** https://github.com/alphagov/govuk-frontend/pull/1056#issuecomment-439381905

---

## Overview

Given the following markup:

```
  <input id="checkbox-bar" name="foo" type="checkbox" value="bar">
  <label for="checkbox-bar">
    Foo
  </label>

  <br>

  <input id="checkbox-baz" name="foo" type="checkbox" value="baz">
  <label for="checkbox-baz">
    Bar
  </label>
```

Focus either checkbox.

This behaviour is also present in Safari Technology Preview release 70.

## Expected behaviour:

Voiceover should announce the label once, e.g.

"Foo, unticked, checkbox"

## Actual behaviour:

Voiceover announces the label twice, e.g.

"Foo, Foo, unticked, checkbox"
